assets/pages/new-contributions-landing/components/ContributionSubmit.jsx

```diff
@@ -25,11 +26,13 @@ type PropTypes = {
 // ----- Render ----- //
 
 function ContributionSubmit(props: PropTypes) {
+  const currency: IsoCurrency = detect(props.countryGroupId);
```

Comment:
This introduces a side-effect into this component (query param lookup), is it possible to use `fromCountryGroupId` from `currency.js` instead?